### ISSUE TITLE
Pagination Bug

### ISSUE THREAD

**Issue by refeals** [OwlCarousel2/OwlCarousel2#2226]
Hello! I'm using OwlTheme for a few days and being mostly satisfied!
However, I've run into something I'd call a bug:

My number of items inside the carousel is dynamic and right now I have 19 items inside, and I'd like to scroll in 5 steps using nav and dots. The problem is when I get to the last page and scroll back, I go from the 4th page to the 2nd.

I looked into the documentation but to no use. Maybe I'm missing something?

Codepen example:
https://codepen.io/refeals/pen/LQmeXr

Thanks in advance!

**Comment by pascalporedda** [comment]
Hi @refeals 

I've been looking at your example and I was wondering myself too. I would consider this a bug, you are not missing anything. If you add another item and have an even number it seems to be good, as far is I've tested it.

**Comment by refeals** [comment]
Yes, I should have pointed that out better. It works fine when you have an even number of items - in this case, 20.

In the last page, we have 15 to 19 as 'active'. When we go back one page, the 'active's are 9 to 14, when, in terms of pagination, we should have 10 to 15.

**Comment by joseantonnio** [comment]
Hi guys,

Any news about this? Have the same here. Happens when the amount of items inside the slider is not a multiple of 5.

**Comment by ghost** [comment]
Hi,

I would suggest adding:     	rewind:true
